src/PrestaShopBundle/Resources/views/Admin/Sell/Address/Blocks/form.html.twig

```diff
@@ -53,95 +53,16 @@
-      }) }}
-
-      {{ ps.form_group_row(addressForm.first_name, {}, {
-        'label': 'First name'|trans({}, 'Admin.Global'),
-        'help' : 'Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' 0-9!<>,;?=+()@#"�{}_$%:'
-      }) }}
-
-      {{ ps.form_group_row(addressForm.last_name, {}, {
-        'label': 'Last name'|trans({}, 'Admin.Global'),
-        'help' : 'Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' 0-9!<>,;?=+()@#"�{}_$%:'
-      }) }}
-
-      {{ ps.form_group_row(addressForm.company, {}, {
-        'label': 'Company'|trans({}, 'Admin.Global'),
-        'help' : 'Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>;=#{}'
-      }) }}
-
-      {{ ps.form_group_row(addressForm.vat_number, {}, {
-        'label': 'VAT number'|trans({}, 'Admin.Orderscustomers.Feature'),
-      }) }}
-
-      {{ ps.form_group_row(addressForm.address1, {}, {
-        'label': 'Address'|trans({}, 'Admin.Global'),
-      }) }}
-
-      {{ ps.form_group_row(addressForm.address2, {}, {
-        'label': 'Address (2)'|trans({}, 'Admin.Global'),
-      }) }}
-
-      {{ ps.form_group_row(addressForm.postcode, {}, {
-        'label': 'Zip/postal code'|trans({}, 'Admin.Global')
-      }) }}
-
-      {{ ps.form_group_row(addressForm.city, {}, {
-        'label': 'City'|trans({}, 'Admin.Global')
-      }) }}
-
-      {{ ps.form_group_row(addressForm.id_country, {'attr': {
-        'class': 'js-customer-country-select',
-        'data-states-url': path('admin_country_states') }}, {
-        'label': 'Country'|trans({}, 'Admin.Global'),
-      }) }}
-
-      <div class="js-address-state-select{% if addressForm.id_state.vars.choices is empty %} d-none{% endif %}">
-        {{ ps.form_group_row(addressForm.id_state, {}, {
-          'label': 'State'|trans({}, 'Admin.Global'),
-        }) }}
-      </div>
-
-      {{ ps.form_group_row(addressForm.phone, {}, {
-        'label': 'Phone'|trans({}, 'Admin.Global')
-      }) }}
-
-      {{ ps.form_group_row(addressForm.phone_mobile, {}, {
-        'label': 'Mobile phone'|trans({}, 'Admin.Global')
-      }) }}
-
-      {{ ps.form_group_row(addressForm.other, {}, {
-        'label': 'Other'|trans({}, 'Admin.Global'),
-        'help' : 'Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>;=#{}'
-      }) }}
-
-      {{ form_rest(addressForm) }}
+      {% block address_form_widget %}
+        {{ form_widget(addressForm) }}
+      {% endblock %}
     </div>
   </div>
 
   <div class="card-footer">
     <div class="d-inline-flex">
-      {% if (displayInIframe is defined and displayInIframe == true) %}
-        <a href="javascript:window.parent.$.fancybox.close();" class="btn btn-outline-secondary">
-      {% else %}
-        <a href="{{ cancelPath|default(path('admin_addresses_index')) }}" class="btn btn-outline-secondary">
-      {% endif %}
+      <a href="{{ path('admin_addresses_index') }}" class="btn btn-outline-secondary">
```

Comment:
@todo need to make sure I didn't break something here

Reply:
Was it for you or for me 😄  ?

Reply:
For me, don't worry abotu it :D